web/app/js/components/GrafanaLink.jsx

```diff
@@ -11,12 +10,7 @@ export default class GrafanaLink extends React.Component {
         to={`/dashboard/db/conduit-deployment?var-namespace=${namespace}&var-deployment=${deployment}`}
         deployment={"grafana"}
         targetBlank={true}>
-        <img
-          src={grafanaicon}
-          width={this.props.size}
-          height={this.props.size}
-          title={`${namespace}/${deployment} grafana dashboard`}
-          alt={`link to ${namespace}/${deployment} grafana dashboard`} />
+        {this.props.name}
```

Comment:
i think we should include an icon to indicate this is opening a new window:
https://fontawesome.com/icons/external-link-alt?style=solid

Reply:
i suggest we do this in a followup, unless it's totally trivial